integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -117,41 +126,64 @@ public void aValidJWTIsReturnedInTheResponse() throws ParseException, IOExceptio
-    }
+        assertFalse(events.isEmpty());
 
-    @And("{int} events are deleted from the audit events SQS queue")
-    public void deleteEventsFromSqsQueue(int messageCount) throws InterruptedException {
-        this.sqs.deleteMatchingMessages(
-                auditEventQueueUrl,
-                messageCount,
-                Collections.singletonMap("/user/session_id", testContext.getSessionId()));
+        for (TestHarnessResponse<AuditEvent<Map<String, Object>>> response : events) {
+            AuditEvent<?> event = response.readAuditEvent();
+            assertEquals("IPV_ADDRESS_CRI_START", event.getEvent());
+            assertEquals(
+                    "deviceInformation", event.getRestricted().getDeviceInformation().getEncoded());
+        }
     }
 
-    private String getDeviceInformationHeader() throws InterruptedException, IOException {
-        final List<Message> startEventMessages =
-                this.sqs.receiveMatchingMessages(
-                        auditEventQueueUrl,
-                        1,
-                        Map.ofEntries(
-                                entry("/event_name", "IPV_ADDRESS_CRI_START"),
-                                entry("/user/session_id", testContext.getSessionId())));
-
-        assertEquals(1, startEventMessages.size());
+    @And("a valid START event is returned in the response without txma header")
+    public void aValidStartEventIsReturnedInTheResponseWithoutTxmaHeader() throws IOException {
+        String responseBody = testContext.getTestHarnessResponseBody();
+        assertEquals(200, testContext.getTestHarnessResponse().httpResponse().statusCode());
+        assertNotNull(responseBody);
+
+        List<TestHarnessResponse<AuditEvent<Map<String, Object>>>> events =
+                objectMapper.readValue(responseBody, new TypeReference<>() {});
+
+        assertFalse(events.isEmpty());
+
+        for (TestHarnessResponse<AuditEvent<Map<String, Object>>> response : events) {
+            AuditEvent<?> event = response.readAuditEvent();
+            assertEquals("IPV_ADDRESS_CRI_START", event.getEvent());
+            assertTrue(
+                    JsonSchemaValidator.validateJsonAgainstSchema(
+                            response.getEvent().getData(), addressStartJsonSchema));
+            assertNull(event.getRestricted());
+        }
+    }
 
-        return objectMapper
-                .readTree(startEventMessages.get(0).body())
-                .at("/restricted/device_information/encoded")
-                .asText();
+    @Then("START TxMA event is validated against schema")
+    public void startTxmaEventValidatedAgainstSchema() throws IOException {
+        String responseBody = testContext.getTestHarnessResponseBody();
+
+        List<TestHarnessResponse<AuditEvent<Map<String, Object>>>> events =
+                objectMapper.readValue(responseBody, new TypeReference<>() {});
+        assertFalse(events.isEmpty());
+
+        boolean checked = false;
```

Comment:
Why does this test have this checked logic but the other do not?

Reply:
Maybe something like this would be better 

```
List<TestHarnessResponse<AuditEvent<Map<String, Object>>>> testHarnessResponses =
                objectMapper.readValue(responseBody, new TypeReference<>() {});
        
        var events =
                testHarnessResponses.stream()
                        .filter(event -> event.getEvent().equals("IPV_ADDRESS_CRI_START"))
                        .collect(Collectors.toList());
        
        assertNotNull(events);
        for (TestHarnessResponse<AuditEvent<Map<String, Object>>> testHarnessResponse : events) {
            AuditEvent<?> event =
                    objectMapper.readValue(
                            testHarnessResponse.getEvent().getData(), AuditEvent.class);
            if (event.getEvent().equalsIgnoreCase("IPV_ADDRESS_CRI_START")) {
                assertTrue(
                        JsonSchemaValidator.validateJsonAgainstSchema(
                                testHarnessResponse.getEvent().getData(), addressStartJsonSchema));
            }
        }
```
Then you don't need to do check logic

Reply:
Maybe we should also check the sessionId in the event is the same as the one here `this.testContext.getSessionId()`